openedx/core/djangoapps/content_staging/models.py

```diff
@@ -8,6 +8,7 @@
 from django.core.exceptions import ValidationError
 from django.db import models
 from django.utils.translation import gettext_lazy as _
+from jsonfield import JSONField
```

Comment:
Shouldn't we just use [`models.JSONField`](https://docs.djangoproject.com/en/4.2/ref/models/fields/#jsonfield)? I don't think we need a third party package for this.

Reply:
Ah oops, I didn't realise that was a real field now. I'll fix it and the migration. Ok if I squash and force push?